content/guides/metadata/0-index.md

```diff
@@ -0,0 +1,62 @@
+---
+rank: 160
+alias_paths:
+  - /docs/work-with-metadata
+---
+
+# Metadata
+
+Metadata allows users and applications to define and store custom data
+associated with files and folders.
+
+<ImageFrame border center>
+
+  ![String field](./metadata-example.png)
+
+</ImageFrame>
+
+Metadata consists of key/value pairs are assigned to a file or a folder.
+For example, an important contract may have the key/value pairs of
+`clientNumber: 820183` and `clientName: bioMedicalCorp`.
+
+## Templates, Fields, Instances, and Cascade Policies
+
+Working with metadata requires a developer to work with a few
+distinct types of resources.
+
+* **Templates:**  A [metadata template][template] describes a re-usable set of
+  key/value pairs that can be assigned to a file. For example, an `invoiceData`
+  template might hold data about an invoice, having a field for the invoice ID
+  as well as the customer ID.
+  * **Fields:**  A [metadata template field][field] describes a specific
```

Comment:
Is the extra spacing intended here for formatting?

Reply:
The extra left indent? Yes, cause fields are part of templates.